### DISCUSSION TITLE
wine ge custom vs wine format

### DISCUSSION THREAD

**Discussion by ghost** [Kron4ek/Wine-Builds#117] [General]
I'm using your wine and proton builds with bin share lib format.  This works perfectly in batocera (thanks!), but seeing a lot of requests for ge builds.  When I extracted them they are split into lib and lib64 and they don't seem to run.  It seems to be they are using different libraries perhaps?

e.g. logs say wine: could not load ntdll.so: libunwind.so.8: cannot open shared object file: No such file or directory etc.  

Was wondering if you were familiar with the differences

**Comment by Kron4ek** [comment]
I'm aware that wine-ge uses two separate directories for 32-bit and 64-bit libraries, but that's not a problem.

> e.g. logs say wine: could not load ntdll.so: libunwind.so.8: cannot open shared object file: No such file or directory etc.

You can solve this problem by installing libunwind into your system, not sure if batocera allows to install packages though.

>  It seems to be they are using different libraries perhaps?

They are compiled in a different build environment, i think this is why my builds do not require libunwind.so.8, but wine-ge does.

**Reply by ghost** [reply]
the base system is immutable except the userdata similar to conty, and there's no package manager since its baked on buildroot.  I've seen some crazy workarounds though.
